custom_components/waste_collection_schedule/waste_collection_schedule/source/wellington_govt_nz.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+from html.parser import HTMLParser
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+from waste_collection_schedule.service.ICS import ICS
+
+
+TITLE = "Wellington City Council"
+DESCRIPTION = "Source for Wellington City Council."
+URL = "https://wellington.govt.nz"
+TEST_CASES = {
+    "Chelsea St": {"streetName": "chelsea street"},  # Friday
+    "Campbell St (ID Only)": {"streetId": "6515"},  # Wednesday
+}
+
+class Source:
+    def __init__(self, streetId= None, streetName= None):
+        self._streetId = streetId
+        self._streetName = streetName
+        self._ics = ICS()
+
+    def get_icon(self, wasteType):
+        if wasteType == "Rubbish Collection":
+            return "mdi:trash-can"
+        if wasteType == "Glass crate":
+            return "mdi:glass-fragile"
+        if wasteType == "Wheelie bin or recycling bags":
+            return "mdi:recycle"
+        return None
+
+    def get_image(self, wasteType):
+        if wasteType == "Rubbish Collection":
+            return "https://wellington.govt.nz/assets/images/rubbish-recycling/rubbish-bag.png"
+        if wasteType == "Glass crate":
+            return "https://wellington.govt.nz/assets/images/rubbish-recycling/glass-crate.png"
+        if wasteType == "Wheelie bin or recycling bags":
+            return "https://wellington.govt.nz/assets/images/rubbish-recycling/wheelie-bin.png"
+        return None
+        
+
+    def fetch(self):
+        # get token
+        if self._streetName:  
+            import json
```

Comment:
move import to top

Reply:
I was keeping it further down as it was only needed if streetName was used. 
I've moved it up the top.